package.json

```diff
@@ -21,22 +21,21 @@
     "node": ">=6"
   },
   "dependencies": {
+    "chalk": "^2.4.2",
     "css": "^2.2.1",
-    "css-selector-parser": "^1.1.0",
```

Comment:
@mirisuzanne I didn't see this being used anywhere? Am I missing something?

Reply:
That may be left over from the original way we handled test-result parsing with selectors rather than comments?